contribute/design/1.x/csi/20190606-csi-volume-provisioning.md

```diff
@@ -371,11 +373,12 @@ metadata:
   namespace: <openebs system namespace>
   labels:
   annotations:
-    openebs.io/config-class: 
+    openebs.io/cstor-volume-config-class: 
     openebs.io/volume-id:
 spec:
   capacity:
   claimRef:
+  replicationFactor:
```

Comment:
lets get this as parameters from storageclass

Reply:
@vishnuitta CSi is getting this from storage class parameters and passing this info via CstorVolumeClaim.